definitions/loosely_coupled_architecture.md

```diff
@@ -1,15 +1,18 @@
 ---
 title: Loosely Coupled Architecture
 status: Feedback Appreciated
-category: concept
+category: Technology
 ---
 ## Loosely Coupled Architecture
 
-A loosely coupled architecture is a system design approach in which an application has little knowledge or reliance on a separate application. Microservices are an example of this architectural approach with a focus on narrowing an application's external dependencies. 
+### What it is
 
-Loosely coupled architecture is the opposite of tightly coupled architecture.
+Loosely Coupled Architecture is a system design approach in which applications are lean, with a single responsibility, without many dependencies, allowing teams to work, deploy, fail and scale independently, increasing business responsiveness. Microservices are an example of this architectural approach with a focus on narrowing an application's external dependencies.
 
-When applications and processes are reliant on each other, the development lifecycle also needs to be intertwined. Updates, scaling, testing, and new features will be implemented against more extensive codebase. The larger the codebase and its dependencies, the more the application can be considered a monolith.
+### Problem it addresses
 
-By decoupling an application, application development will be quicker since external dependencies will not hamper it. At an organizational level, teams can be structured around their competency, focusing on the specific application.
+When applications and processes are reliant on each other, the development lifecycle also needs to be intertwined. Updating, scaling, testing, and adding new features against an extensive codebase increases technical debt and decreases feasibility. The larger the codebase and its dependencies, becomes more or less like Tight Coupled Architectures (aka Monoliths), which are brittle, hard to change, test, and release.
```

Comment:
This last sentence, "The larger the codebase and its dependencies, becomes more or less like Tight Coupled Architectures (aka Monoliths), which are brittle, hard to change, test, and release." Doesn't make sense. It looks like it's two somewhat related sentences jammed together.

Reply:
Okay,
Thanks for the suggestion.
I will remove the last line.